lib/screenplay_web/controllers/auth_controller.ex

```diff
@@ -18,11 +24,12 @@ defmodule ScreenplayWeb.AuthController do
     Plug.Conn.delete_session(conn, :previous_path)
 
     conn
+    |> configure_session(drop: true)
     |> Guardian.Plug.sign_in(
       ScreenplayWeb.AuthManager,
       username,
-      %{roles: roles},
-      ttl: {session_ttl_hours, :hours}
+      %{auth_time: auth_time, roles: roles},
+      ttl: {1, :minute}
```

Comment:
Does this only work because we know our frontend always makes a request to the server at least once a minute? (just trying to get an idea of how this would look under different conditions, like if this were a plain server-rendered app)

Reply:
Correct. My understanding is that the value here can be anything as long as it's <= the idle time we configure. Because we are guaranteed to refresh every 4 seconds, 1 minute will always be enough.